src/main/webapp/app/overview/courses.component.ts

```diff
@@ -38,7 +38,7 @@ export class CoursesComponent implements OnInit {
     }
 
     loadAndFilterCourses() {
-        this.courseService.findAll().subscribe(
+        this.courseService.findAllForDashboard().subscribe(
```

Comment:
This change broke something, no courses are displayed anymore 
![image](https://user-images.githubusercontent.com/29718932/82362271-613e9d80-9a0c-11ea-8f42-c08ae9aae4f5.png)

Reply:
same for me

Reply:
This method does not exist on courseService

Reply:
@stefanwaldhauser `courseService` is an instance of `CourseManagementService`.

Reply:
Ah got it thank you :)

Reply:
Where do I see this in typescript @madwau ?

Reply:
You can see it in the constructor where this dependency is injected.

Reply:
haha oh man i see what you mean know :) I thought you meant something with inheritance. The variable name should probably changed to courseManagementService though

Reply:
I'm guessing `courseManagementService` was seen as too verbose, not sure though.

Reply:
I am currently working on a fix. The problem is that a visible quiz has no due date, but this was expected for the sorting of the exercise in the course overview